client/jetpack-connect/remote-credentials.js

```diff
@@ -159,12 +150,7 @@ export class OrgCredentialsForm extends Component {
 	}
 
 	formHeader() {
-		return (
-			<div>
-				{ this.getHeaderImage() }
-				{ this.renderHeadersText() }
-			</div>
-		);
+		return <div>{ this.renderHeadersText() }</div>;
```

Comment:
The logo rebased opened a bit of a can of worms by causing these lines to be modified 😅 

The wrapper div looks unnecessary at this point.

`renderHeadersText` looks like a good candidate to just be inlined JSX. `formHeader` could be inlined as well. These methods hold little value.

Reply:
yes, this is true. Could have looked closer 👍